targets/targets.json

```diff
@@ -969,7 +969,7 @@
         "extra_labels": ["STM", "STM32F7", "STM32F756", "STM32F756xG", "STM32F756ZG"],
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "IAR"],
         "default_toolchain": "ARM",
-        "macros": ["TRANSACTION_QUEUE_SIZE_SPI=2", "USBHOST_OTHER", "MBEDTLS_CONFIG_HW_SUPPORT"],
+        "macros": ["TRANSACTION_QUEUE_SIZE_SPI=2", "USBHOST_OTHER", "MBEDTLS_CONFIG_HW_SUPPORT", "MBEDTLS_MD5_C"],
```

Comment:
@adustm I am sorry if I didn't explaoin correct. user defined features should be defined by the user in mbed_app.json or the uer defined config file. putting these types of definitions to targets.json will define MD5 for all of these targets, regardless of user's decision